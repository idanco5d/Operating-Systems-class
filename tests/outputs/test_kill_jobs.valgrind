==31502== Memcheck, a memory error detector
==31502== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==31502== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==31502== Command: /mnt/c/Users/illus/os/hw/1/smash
==31502== Parent PID: 31497
==31502== 
==31502== 
==31502== FILE DESCRIPTORS: 4 open (3 std) at exit.
==31502== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_kill_jobs.valgrind
==31502==    <inherited from parent>
==31502== 
==31502== 
==31502== HEAP SUMMARY:
==31502==     in use at exit: 0 bytes in 0 blocks
==31502==   total heap usage: 199 allocs, 199 frees, 88,445 bytes allocated
==31502== 
==31502== All heap blocks were freed -- no leaks are possible
==31502== 
==31502== For lists of detected and suppressed errors, rerun with: -s
==31502== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
