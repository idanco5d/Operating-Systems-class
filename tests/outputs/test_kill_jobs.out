smash> submitted by Eran
smash> smash> smash> smash> [1] ./printSignals.exe & : 9446 0 secs
[2] sleep 200& : 9447 0 secs
[3] sleep 300& : 9448 0 secs
smash> signal number 9 was sent to pid 9447
smash> smash> [1] ./printSignals.exe & : 9446 1 secs
[3] sleep 300& : 9448 1 secs
[4] sleep 400& : 9496 0 secs
smash> smash> smash> smash> smash> smash> signal number 9 was sent to pid 9496
smash> signal number 5 was sent to pid 9446
smash> received signal number: 5
smash> [1] ./printSignals.exe & : 9446 4 secs
[3] sleep 300& : 9448 4 secs
[4] sleep 500& : 9748 0 secs
smash> smash: sending SIGKILL signal to 3 jobs:
9446: ./printSignals.exe &
9448: sleep 300&
9748: sleep 500&
received signal number: 15
